docs/kb/rules/understand-severities.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Understand how rule severity is determined.
+tags:
+  - Rules
+  - Semgrep Registry
+---
+
+# How does Semgrep assign severity levels to rules?
+
+## Semgrep Code
+
+Semgrep Code rules have one of three severity levels: ERROR (High), WARNING (Medium), or INFO (Low). The severity indicates how critical the issues are that a rule potentially detects.
+
+The severity of a given rule is assigned by the rule author. For custom and third-party rules, the rule author's assignment is the source of truth.
+
+As a best practice, severity for Semgrep Registry rules in the `security` category should be assigned by evaluating the combination of [likelihood](/docs/contributing/contributing-to-semgrep-rules-repository/#likelihood) and [impact](/docs/contributing/contributing-to-semgrep-rules-repository/#impact). 
+
+## Semgrep Supply Chain 
+
+Semgrep Supply Chain rules have one of four severity levels: Critical, High, Medium or Low. The score assigned to the CVE using the [Common Vulnerability Scoring System (CVSS) score])(https://nvd.nist.gov/vuln-metrics/cvss), or the severity value set by the GitHub Advisory Database, determines the severity in Semgrep Supply Chain. If a vulnerability is given a 9.0 or higher it is assigned CRITICAL.
```

Comment:
I feel like we could either go without defining what Critical's score range is, or if we do, do it for all the other severities as well. Also I think "CRITICAL" should just be "Critical", like the first sentence.

Reply:
yeah you're right, the severity writing should be consistent.

I'll see if I can find out more about the other severities before we merge this :) I only got the 9+ score info from what we had internally.

Reply:
I was able to verify that when providing that example we were also referencing the same scoring rubric shown in the linked article, that just happened to be the level called out in the comment. I changed the wording to indicate that it's an example.